loader/loader_linux.c

```diff
@@ -271,7 +268,7 @@ VkResult linux_read_sorted_physical_devices(struct loader_instance *inst, uint32
             sorted_device_info[index].device_id = dev_props.deviceID;
 
             bool device_is_1_1_capable =
-                VK_API_VERSION_MAJOR(dev_props.apiVersion) == 1 && VK_API_VERSION_MINOR(dev_props.apiVersion) >= 1;
+                loader_check_version_meets_required(loader_combine_version(1, 1, 0), loader_make_version(dev_props.apiVersion));
```

Comment:
Nit, you could make one "combined_version" and use it here and line 239 above without re-generating it.

Reply:
Added a macro LOADER_VERSION_1_1_0 which is just a define for `loader_combine_version()`